nixpkgs/nixpkgs.bzl

```diff
@@ -99,11 +99,9 @@ def _nixpkgs_package_impl(ctx):
   # Build a forest of symlinks (like new_local_package() does) to the
   # Nix store.
 
-  find_path = ctx.which("find")
-  if find_path == None:
-    fail("Could not find the 'find' command. Please ensure it is in your PATH.")
+  find_path = _executable_path("find", ctx)
 
-  res = ctx.execute(["find", output_path, "-maxdepth", "1"])
+  res = ctx.execute([find_path, output_path, "-maxdepth", "1"])
```

Comment:
This actually fixes a bug in some situations, `find_path` should have been used here all along.